packages/rule-typescript-config-is-valid/tests/fixtures/noconfig/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+# 1.0.0 (February 8, 2018)
```

Comment:
It's a bit weird to have this as a fixture. Put other content that explains the purpose of this file.